src/components/note-containers/note-containers.css

```diff
@@ -6,7 +6,29 @@
 }
 
 .note-container {
-  width: 25rem;
-  height: 8rem;
+  width: 400px;
+  height: 120px;
+  margin-bottom: 10px;
   border: 1px solid black;
+  background-color: thistle;
+}
+
+.delete-btn {
+  width: 20px;
+  position: relative;
```

Comment:
if the text of `<div>{note.id}</div>` will be too long, it will be under the button. Probably wrapping the header of the box into some div and marking it as `flex` would be better